ui/DesignSystem/Component/Wallet/TransactionListItem.svelte

```diff
@@ -0,0 +1,129 @@
+<script>
+  export let tx = null;
+  import {
+    headerIcon,
+    formatMessage,
+    formatSubject,
+    iconState,
+    iconProgress,
+    statusText,
+    subjectAvatarShape,
+  } from "../../../src/transaction.ts";
+  import { microRadToRad } from "../../../src/currency.ts";
+
+  import { Rad } from "../../Component";
+  import { Avatar, Caption, Icon, Text, Title } from "../../Primitive";
+
+  const subject = formatSubject(tx.messages[0]);
+  console.log(subject);
+
+  const formatDate = (timestamp, option) => {
```

Comment:
If we move this logic into `transaction.ts`, we can unit test it 🎉

Reply:
👍